### PR TITLE
Use meta for rates quality mode

### PR DESCRIPTION
**Pull request by martaiborra**

### CODE REVIEW

File: tests/test_meta.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import pytest
+from PIL import Image
+from pathlib import Path
+
+
+import blosc2
+
+project_dir = Path(__file__).parent.parent
+@pytest.mark.parametrize('image', [project_dir / 'examples/kodim23.png', project_dir / 'examples/MI04_020751.tif'])
+@pytest.mark.parametrize('meta', [1 * 10, 2 * 10, 5 * 10, 8 * 10, 10 * 10])
```
Comment (FrancescAlted) [inline tests/test_meta.py:19]: Can you add a test for 20 * 10?  I'd like to see that 200 is not (wrongly) interpreted as negative (128 - 200 = -72).

**Review by FrancescAlted — APPROVED** [review]
I have just a comment.  Other than that, this looks good to me.
